Assets/UI/Popups/mappinlistpanel.xml

```diff
@@ -2,7 +2,7 @@
 
 <Context Style="FontNormal16">
   <!-- Map Pin List Panel -->
-  <Grid Anchor="L,B" ID="MapPinPanel" Size="320,119" Texture="Tracker_OptionsBacking" SliceCorner="55,61" SliceSize="1,1" SliceTextureSize="121,119" ConsumeAllMouse="1" MinSize="121,119" AutoSize="V" AutoSizePadding="0,15">
+  <Grid Anchor="L,B" ID="MapPinPanel" Size="320,auto" Texture="Tracker_OptionsBacking" SliceCorner="55,61" SliceSize="1,1" SliceTextureSize="121,119" ConsumeAllMouse="1" MinSize="121,119" AutoSize="V" AutoSizePadding="0,15">
```

Comment:
Autosize="V" should be removed

Reply:
In line 32, Size="200,30" AutoSize="V" should maybe be changed to Size="200,auto" (as done in Civ VI Fall Update) even if the code is commented out - Just for the case where the code gets reactivated in a future release

Reply:
Done those two and did a commit.